tests/ROCrate/Common.fs

```diff
@@ -0,0 +1,38 @@
+module Tests.Common
```

Comment:
Fine for now, but when everything is settled we should move it to TestingUtils project